src/components/Charts/ChartVaccinations.tsx

```diff
@@ -224,18 +231,16 @@ const VaccinationLines: React.FC<{
                   barOpacity={1}
                   params={params}
                 />
-                {/* Vaccination data from the backend is capped at 95%. 
-                Use a dashed line for all capped data. */}
                 <ChartSeries
                   key={`series-chart-${label}`}
-                  data={data.filter(d => d.y >= 0.95)}
+                  data={data.filter(d => d.y >= VACCINATION_PERCENTAGE_CAP)}
                   x={getXPosition}
                   y={getYPosition}
                   type={type}
                   yMax={innerHeight}
                   barWidth={0}
                   barOpacity={1}
-                  params={{ ...params, ...dashedLineParam }}
+                  params={{ ...params, ...{ strokeDasharray: '1, 6' } }}
```

Comment:
nit: I think this can just be `{ ...params, strokeDasharray: '1, 6' }`

Reply:
it didn't seem to like that :( it threw some errors:

```JSX spread child must be an array type.ts(2609)``` and ```Left side of comma operator is unused and has no side effects.ts(2695)```

Reply:
Oh, I should have said `{{ ...params, strokeDasharray: '1, 6' }}`

The outer `{ }` are used to insert a JavaScript expression inside the JSX block.  The inner `{  }` are creating a JS object literal.  So that needs to stay the same.  The thing I was pointing out that could be simplified was the `...{ strokeDasharray: '1, 6' }`.  That was creating an object literal with a single`strokeDasharray` property and then merging it into the parent object with the spread operator (`...`), which is unnecessary. You can just define the `strokeDasharray` property directly on the parent object.

That said, it turns out when you do that, TypeScript apparently gets more smart / picky about the types here and complains:
![image](https://user-images.githubusercontent.com/206364/148868529-37482bc1-f3c3-4909-af94-e737ef0373f3.png)

And that's my fault.  I think I was wrong [here](https://github.com/covid-projections/covid-projections/pull/4983/files/00da19cf1c425ef744e8ab11d6be4f5fa68041ef#r781638356).  You need to add `strokeDasharray` back to `SeriesParams` in `src/components/Explore/interfaces.ts`.  Sorry about that!

Reply:
Ohh I see, and no worries at all! Thanks for all the explanations, they've really been a big help 🙏

Reply:
updated this in 362d2890d1b452333dc3a91ca24f1bb4211c4211 and am going to 🚢